src/ContentRepository/Preview/DocumentPreviewProvider.cs

```diff
@@ -705,7 +723,7 @@ public virtual IEnumerable<Content> GetPreviewImages(Content content)
         /// <param name="content">The content that has preview images.</param>
         public virtual IEnumerable<Content> GetExistingPreviewImages(Content content)
         {
-            if (content == null || !this.IsContentSupported(content.ContentHandler) || !HasPreviewPermission(NodeHead.Get(content.Id)))
+            if (content == null || !this.IsPreviewEnabled(content.ContentHandler) || !HasPreviewPermission(NodeHead.Get(content.Id)))

```

Comment:
This will prevent the client from loading existing preview images, which is not what we want. Existing preview images should be displayed.